src/test/java/Homework17.java

```diff
@@ -0,0 +1,139 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest{
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+//        // Pre-condition
+//        //      Added ChromeOptions argument below to fix websocket error
+//        ChromeOptions options = new ChromeOptions();
+//        options.addArguments("--remote-allow-origins=*");
+//
+//        WebDriver driver = new ChromeDriver(options);
+//        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+//
+//        //Steps
+//        //Step 1
+//        String url = "https://bbb.testpro.io/";
+//        driver.get(url);
+//        //Step 2
+//        WebElement emailField = driver.findElement(By.cssSelector("input[type='email']"));
+//        emailField.click();//not needed
+//        emailField.clear();
+//        emailField.sendKeys("demo@class.com");
+//
+//        WebElement passwordField = driver.findElement(By.cssSelector("input[type='password']"));
+//        passwordField.click();//not needed
+//        passwordField.clear();
+//        passwordField.sendKeys("te$t$tudent");
+//
+//        WebElement submit = driver.findElement(By.cssSelector("button[type='submit']"));
+//        submit.click();
+//
+//        WebElement avatarIcon = driver.findElement(By.cssSelector("img[class='avatar']"));
+//        Assert.assertTrue(avatarIcon.isDisplayed());
+//        line 14 - 36 will be replaced with the ff login code below:
+
+        //lets try first if our login works, then we can proceed to implement the next steps
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.